crates/extensions/c8y_http_proxy/src/credentials.rs

```diff
@@ -34,10 +24,11 @@ pub struct C8YJwtRetriever {
 }
 
 impl C8YJwtRetriever {
-    pub fn builder(mqtt_config: mqtt_channel::Config) -> JwtRetrieverBuilder<C8YJwtRetriever> {
-        JwtRetrieverBuilder::new(C8YJwtRetriever {
-            mqtt_config: mqtt_config.with_subscriptions(TopicFilter::new_unchecked("c8y/s/dat")),
-        })
+    pub fn builder(
+        mqtt_config: mqtt_channel::Config,
+    ) -> ServerActorBuilder<C8YJwtRetriever, Sequential> {
+        let server = C8YJwtRetriever { mqtt_config };
+        ServerActorBuilder::new(server, &ServerConfig::default(), Sequential)
```

Comment:
The default`ServerConfig` is "concurrent". Didn't you want to override that here for a `Sequential` actor?

Reply:
The `ServerConfig` is neither concurrent nor sequential. It just provides numbers to be used by a concrete implementation.

I agree that this is misleading. One option to fix that is to add the kind (i.e. `Sequential` or `Concurrent`) to `ServerConfig`. I will try to do that.